### PR TITLE
remove content for TFJob dashboard

### LINKED ISSUE #1091: Remove mention of TFJob dashboard from the docs
We've stopped supporting the tfjob dashboard, so it is entirely possible that it is broken in Kubeflow v0.6+.

Remove mention of the TFJob dashboard from the docs:
* https://www.kubeflow.org/docs/components/training/tftraining/#accessing-the-tfjob-dashboard
* https://www.kubeflow.org/docs/other-guides/accessing-uis/#next-steps

### PR DESCRIPTION
**Pull request by kbhawkey**
Closes #1091.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1120)

### CODE REVIEW

**Comment by kbhawkey** [comment]
preview pages:
https://deploy-preview-1120--competent-brattain-de2d6d.netlify.com/docs/components/training/tftraining/

https://deploy-preview-1120--competent-brattain-de2d6d.netlify.com/docs/other-guides/accessing-uis/

File: content/docs/other-guides/accessing-uis.md [hunk 1]
```diff
@@ -11,12 +11,11 @@ instructions on how to connect to them.
 
 The Kubeflow UIs include the following:
 
-* A central **Kubeflow** UI for navigation between the Kubeflow applications.
-* **Pipelines** for a Kubeflow Pipelines dashboard
-* **Notebook Servers** for Jupyter notebooks.
-* **Katib** for hyperparameter tuning.
-* **Artifact Store** for tracking of artifact metadata.
-* **tf-operator** for a TFJob dashboard.
+* Central UI for navigation
```
Comment (kbhawkey) [inline content/docs/other-guides/accessing-uis.md]: Looks like I did not pay attention to my rebase.

**Comment by sarahmaddox** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**